typescript/forms.d.ts

```diff
@@ -800,8 +800,6 @@ export type Form = {
   publishStartDate?: string
   /** The date and time (in ISO format) a form becomes unavailable. */
   publishEndDate?: string
-  /** Whether or not the Form is an Info Page. */
```

Comment:
Need to also remove `InfoPageElement`

Reply:
When we remove the `InfoPageElement`, can we please change the `FormFormElement['type']` to  `'form' | 'infoPage'`.

Reply:
Why do we need that?

Reply:
There is data in S3 that we cannot practically update with `'infoPage'` elements and if we remove the type, we need to have a bunch of `@ts-expect-error` statements in all the places that render historical data.